### PR TITLE
[CoreBundle] Fixed anonymous subscription with validation

### PR DESCRIPTION
**Pull request by ptsavdar**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Fixed tickets | #1601 

Fixes anonymous user registration via workspace form with "validation" option. Shows message to user that his account has been created however he won't be able to access the workspace unless an administrator validates his enrollment.

### CODE REVIEW

File: main/core/Resources/translations/platform.de.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ accept: Accepter
 accessible_from: 'Accessible à partir du'
 accessible_until: 'Accessible jusqu''au'
 account_created: 'Les informations de votre compte vous ont été envoyées par mail (il est possible qu''elles arrivent dans votre dossier dédié aux spams).'
+account_created_awaiting_validation: 'Votre compte a été crée avec succès. Toutefois, il ne serait possible d''acceder à l''espace de travail qu''après la validation par un administrateur.'
```
Comment (Elorfin) [inline main/core/Resources/translations/platform.de.yml]: `il ne sera possible, d''accéder`

File: main/core/Resources/translations/platform.en.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ accept: Accept
 accessible_from: 'Accessible from'
 accessible_until: 'Accessible until'
 account_created: 'Account created.'
+account_created_awaiting_validation: 'Your account has been created successfully. However you will not be able to access the workspace unless an administrator validates your account.'
```
Comment (Elorfin) [inline main/core/Resources/translations/platform.en.yml]: `Your account has been successfully created. `

File: main/core/Resources/translations/platform.fr.yml [hunk 3]
```diff
@@ -4,6 +4,7 @@ accept: Accepter
 accessible_from: 'Accessible à partir du'
 accessible_until: 'Accessible jusqu''au'
 account_created: 'Les informations de votre compte vous ont été envoyées par courriel. (Il est possible qu''elles arrivent dans votre dossier dédié aux pourriels).'
+account_created_awaiting_validation: 'Votre compte a été crée avec succès. Toutefois, il ne serait possible d''acceder à l''espace de travail qu''après la validation par un administrateur.'
```
Comment (Elorfin) [inline main/core/Resources/translations/platform.fr.yml]: `il ne sera possible, d''accéder`

File: main/core/Resources/translations/platform.nl.yml [hunk 4]
```diff
@@ -4,6 +4,7 @@ accept: Aanvaarden
 accessible_from: Toegankelijk vanaf
 accessible_until: Toegankelijk tot
 account_created: De informatie van uw rekening is u per email gestuurd. (Kijk eventueel of die niet in uw spams zit).
+account_created_awaiting_validation: 'Votre compte a été crée avec succès. Toutefois, il ne serait possible d''acceder à l''espace de travail qu''après la validation par un administrateur.'
```
Comment (Elorfin) [inline main/core/Resources/translations/platform.nl.yml]: `il ne sera possible, d''accéder`

File: main/core/Controller/Tool/WorkspaceParametersController.php [hunk 5]
```diff
@@ -337,7 +336,14 @@ public function anonymousSubscriptionAction(Workspace $workspace)
         if ($form->isValid()) {
             $user = $form->getData();
             $this->userManager->createUser($user);
-            $this->workspaceManager->addUserAction($workspace, $user);
+            if ($workspace->getRegistrationValidation() === true) {
```
Comment (Elorfin) [inline main/core/Controller/Tool/WorkspaceParametersController.php]: `if ($workspace->getRegistrationValidation())` would suffice.

Comment (ptsavdar) [inline main/core/Controller/Tool/WorkspaceParametersController.php]: Done

File: main/core/Resources/translations/platform.de.yml (hunk 1 above)
Comment (ptsavdar) [inline main/core/Resources/translations/platform.de.yml]: Done

File: main/core/Resources/translations/platform.en.yml (hunk 2 above)
Comment (ptsavdar) [inline main/core/Resources/translations/platform.en.yml]: Done

File: main/core/Resources/translations/platform.fr.yml (hunk 3 above)
Comment (ptsavdar) [inline main/core/Resources/translations/platform.fr.yml]: Done

File: main/core/Resources/translations/platform.nl.yml (hunk 4 above)
Comment (ptsavdar) [inline main/core/Resources/translations/platform.nl.yml]: And done ;)

**Comment by ptsavdar** [comment]
@Elorfin I fixed the translations

**Pull request merged**